SECURITY.md

```diff
@@ -7,7 +7,9 @@ currently being supported with security updates.
 
 | Version | Supported          |
 | ------- | ------------------ |
-| 21.1.x   | :white_check_mark: |
+| 23.3.x   | :white_check_mark: |
+| 23.2.x   | :white_check_mark: |
+| 23.1.x   | :white_check_mark: |
```

Comment:
I wonder if it would make sense to list the versions expected EoL and when previous versions went EoL.

Reply:
For sure it will not hurt, and it is security related so folks will appreciate.